docs/snippets/common/storybook-main-add-sass-config.js.mdx

```diff
@@ -3,18 +3,13 @@
 
 import * as path from 'path';
 
-// Export a function. Accept the base config as the only param.
 export default {
+  framework: '@storybook/vue3-vite',
   stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
-  addons: [
-    '@storybook/addon-links',
-    '@storybook/addon-essentials',
-    '@storybook/addon-interactions',
-  ],
   webpackFinal: async (config, { configType }) => {
```

Comment:
This is used as the example for `config.webpackFinal`, but I believe this particular config is now covered by addon-styling. Is there a better example to use?

Reply:
Going to replace this with a path aliasing example.